sap_document_classification_client/dc_api_client.py

```diff
@@ -512,7 +508,8 @@ def _validate_results(results, error_message):
             raise Exception(error_message + ': ' + str(failed_results))
 
     @staticmethod
-    def _find_files(directory, pattern):
-        rule = re.compile(fnmatch.translate(pattern), re.IGNORECASE)
-        return [os.path.join(directory, name) for name in os.listdir(directory) if rule.match(name)]
+    def _find_files(directory):
+        return [os.path.join(directory, name) for name in os.listdir(directory)
+                if name.lower().endswith(FILE_EXTENSIONS_FOR_FOLDER_UPLOAD)]
```

Comment:
I would rather check that the extension is different form json and not maintain the supported file extensions list.